WordPress/theme-experiments
# Bring Parisienne up-to-date with Gutenberg 7.9

This PR makes a few changes: 

- Implements the `tagName` [attribute](https://github.com/WordPress/gutenberg/pull/20218) on group blocks so the markup generated by block templates is semantic: https://github.com/WordPress/theme-experiments/commit/8f23a85c99cae77c5a2373c75f819f714b40acf4
- Takes starter content out of demo templates and makes use of [block patterns](https://developer.wordpress.org/block-editor/developers/block-api/block-patterns/) where relevant: https://github.com/WordPress/theme-experiments/commit/0735d0e9a9ae013349780ace38e580c9ebcebdea
- Addresses a few stylistic bugs with the header and cover block due to changes in GB:
https://github.com/WordPress/theme-experiments/commit/dfde44b224628fed584b6a4583c2bf3dd6514941

How to test:

- Follow the instructions to spin up a new site, activate Gutenberg + FSE, and the theme 
- Try inserting a block pattern
- Ensure that the starter pages were created properly
